app/assets/stylesheets/govuk_publishing_components/lib/_print_support.scss

```diff
@@ -25,3 +25,48 @@
     }
   }
 }
+
+// Two helper classes to ensure that all printed links
+// receive consistent formatting. Most links inherit
+// styles from govuk-frontend, including the printing
+// of the href that a link contains. This makes printed
+// links more useful, but they can take up a lot of space
+// on the page in the default font size and will print
+// in blue by default. These helpers set the printed colour
+// to black, reduce the font size, and improve the layout
+// of the printed href.
+//
+// .gem-print-link
+// ----------------------------------------------------------
+// Typically this class will be applied to existing
+// elements that have the `govuk-link` class but it can
+// also be used on other link elements.
+//
+// .gem-print-links-within
+// ----------------------------------------------------------
+// A variation of the previous print style, to be used on
+// parent elements that contain links with the `govuk-link`
+// class. This is typically only required for views where
+// the `govuk-link` element is not exposed in the markup
+// of the view as it's coming from a self-contained helper.
+
+// stylelint-disable declaration-no-important
+@include govuk-media-query($media-type: print) {
+  .gem-print-link,
+  .gem-print-links-within .govuk-link {
+    &,
```

Comment:
What does this line do?

Reply:
Do you mean the & on line 57? This is a 'failsafe' to ensure consistency between directly styled links such as `a {rules}` and links styled only with pseudo selectors `a:link {rules}`, because they don't behave the same unless you set them all.

Reply:
Also, I've changed line 59 from `&:link:visited` to just `&:visited` as the simpler form seems fine in all modern browsers. For a long time I used the doubled-up form to ensure that old IE applied link styles consistently and it's an ingrained habit that I must stop doing by default...!

Reply:
Thanks.